scripts/apps/archive/controllers/UploadController.js

```diff
@@ -79,15 +80,37 @@ export function UploadController($scope, $q, upload, api, archiveService, sessio
         $scope.enableSave = true;
     };
 
+    var getErrorMessage = function(type) {
+        var errorMessage = 'Only the following files are allowed: ';
+        var separator = '';
+
+        if ($scope.allowPicture && type !== 'image') {
+            errorMessage += separator + type;
+            separator = ' and ';
+        }
+
+        if ($scope.allowPicture && type !== 'video') {
+            errorMessage += separator + type;
+            separator = ' and ';
+        }
+
+        if ($scope.allowPicture && type !== 'audio') {
+            errorMessage += separator + type;
+            separator = ' and ';
+        }
+
+        return gettext(errorMessage);
```

Comment:
it's better to translate it separately, otherwise this would generate a lot of string to translate, but won't be picked up by extractor